main.py

```diff
@@ -67,11 +65,11 @@ def main():
 
             for scenario_id, scenario_params in enumerate(scenario_params_list):
 
+                logger.info(f"Scenario {scenario_id + 1}/{len(scenario_params_list)}")
```

Comment:
Naive question: is there any possible case where the len of `scenario_params_list` could be equal to 0?

Reply:
Hmm, no I don't think so. It would mean launching the program with a config file containing no scenario: doesn't make sense.